.vscode/launch.json

```diff
@@ -10,7 +10,6 @@
 			"args": ["--extensionDevelopmentPath=${workspaceFolder}/editors/vscode" ],
 			"sourceMaps": true,
 			"outFiles": ["${workspaceFolder}/editors/vscode/dist/*.js"],
-			"preLaunchTask": "watch",
```

Comment:
https://github.com/microsoft/vscode/issues/180641, vscode can't prelaunch an esbuild-watch task, remove this line to avoid hanging.